packages/react/src/components/TooltipIcon/TooltipIcon.js

```diff
@@ -13,6 +13,7 @@ import { composeEventHandlers } from '../../tools/events';
 import { keys, matches } from '../../internal/keyboard';
 import toggleClass from '../../tools/toggleClass';
 import { usePrefix } from '../../internal/usePrefix';
+import useIsomorphicEffect from '../../internal/useIsomorphicEffect';
 
 const getInstanceId = setupGetInstanceId();
 const TooltipIcon = ({
```

Comment:
We actually don't even export this component anymore, and it's not in the storybook. I think we forgot to remove the files after v11.

Reply:
https://github.com/carbon-design-system/carbon/blob/607b441688769e4c0311e8ddc00f5ba1f51e4c14/docs/migration/v11.md#tooltips:~:text=TooltipIcon%20has%20been%20removed%2C%20use%20IconButton%20instead

Reply:
I opened an issue https://github.com/carbon-design-system/carbon/issues/12824